lib/manageiq/api/common/users.rb

```diff
@@ -0,0 +1,70 @@
+module ManageIQ
+  module API
+    module Common
+      module Users
+        IDENTITY_KEY = 'x-rh-identity'.freeze
+        VALID_KEYS = %w(account_number user username email is_org_admin)
```

Comment:
@syncrou Do we need this list if we decode it we get back a hash and when the user asks for a key if it exists in the hash we give the value back else we raise a key not found error.